modules/dhcp_isc/isc_file_parser.rb

```diff
@@ -152,6 +153,8 @@ def parse_record_options text
           options[:nextServer] = ns
         when /^supersede server.filename\s+=\s+"(\S+)"/
           options[:filename] = $1
+        when /^supersede host-name\s+=\s+"(\S+)"/
```

Comment:
Not sure, I follow. Do you mean an option without the "supersede"?

Something like this?
```
host wiki {
  hardware ethernet 00:00:0e:d2:da:be;
  fixed-address 192.168.2.5;
  option host-name "wiki";
}
```